lib/httpd_configmap_generator/active_directory.rb

```diff
@@ -79,6 +80,26 @@ def domain
 
     private
 
+    def configure_sssd
+      info_msg("Configuring SSSD Service")
+      sssd = Sssd.new(opts)
+      sssd.load(SSSD_CONFIG)
+      sssd.configure_domain(domain)
+      sssd.section("domain/#{domain}")["ad_server"] = opts[:ad_server] if opts[:ad_server].present?
+
+      sssd.section("sssd")["domains"] = domain
+      sssd.section("sssd")["default_domain_suffix"] = domain
+
+      sssd.add_service("pam")
+      sssd.section("pam")["default_domain_suffix"] = domain
+
+      sssd.configure_ifp
+      sssd.section("ifp")["default_domain_suffix"] = domain
```

Comment:
Alberto, I do not believe `default_domain_suffix` is needed in the `pam` and `ifp` sections. On my IPA test client setup it actually produces the below error in the sssd logs.

Please check if this error is in your sssd logs in your AD test environment when setting default_domain_suffix in the `pam` and `ifp` sections and that everything works correctly when removed.

> /var/log/sssd/sssd.log:(Tue Oct 24 08:48:00:133614 2017) [sssd] [sss_ini_call_validators] (0x0020): [rule/allowed_pam_options]: Attribute 'default_domain_suffix' is not allowed in section 'pam'. Check for typos.
> /var/log/sssd/sssd.log:(Tue Oct 24 08:48:00:133885 2017) [sssd] [sss_ini_call_validators] (0x0020): [rule/allowed_ifp_options]: Attribute 'default_domain_suffix' is not allowed in section 'ifp'. Check for typos.

Reply:
Besides this one issue ever thing else `LGTM` 👍

Reply:
Thanks @jvlcek for the 👀 , I'll update and retest.

Reply:
Update tested fine and pushed.  While it worked fine, I was getting the error in the sssd.log same as you.  We should update http://manageiq.org/docs/reference/latest/auth/active_directory#configure-sssd accordingly for completeness.  Thanks for the help